packaging/greenboot/microshift-running-check.sh

```diff
@@ -119,9 +119,22 @@ for i in ${!PODS_NS_LIST[@]}; do
 done
 
 # Verify that MicroShift core pods are not restarting
+declare -A pidNamespace
 for i in ${!PODS_NS_LIST[@]}; do
     CHECK_PODS_NS=${PODS_NS_LIST[$i]}
 
     echo "Checking pod restart count in the '${CHECK_PODS_NS}' namespace"
-    namespace_pods_not_restarting ${CHECK_PODS_NS}
+    namespace_pods_not_restarting ${CHECK_PODS_NS} &
+    pid=$!
+    pidNamespace["${pid}"]="${CHECK_PODS_NS}"
+done
+
+set +e
+for pid in "${!pidNamespace[@]}"; do
+    wait "${pid}"
```

Comment:
Can we just wait for all the sub-shells to exit? We should not need to disable exit-on-error - let's just fail the script if the background shell returns an error?

Reply:
We should get an output about the failed background process, right?

Reply:
> We should get an output about the failed background process from bash, right?

You mean something like this?
```
[1]+  Exit 1 
```
I think it's only in interactive sessions